src/parser-json.js

```diff
@@ -58,10 +57,14 @@ function toBabylon(node) {
       return Object.assign(result, {
         elements: node.children.map(toBabylon)
       });
-    case "literal":
-      return Object.assign(result, {
-        rawValue: node.rawValue
+    case "literal": {
+      const babylonExpression = require("./parser-babylon")(node.rawValue, {
```

Comment:
This is going to embed the entire babylon parser in the build step.

There's only a handful of json types, can you just create them from scratch? This way you don't have to do all this options passing